Document/0x05g-Testing-Network-Communication.md

```diff
@@ -99,7 +99,7 @@ Dynamic analysis requires an interception proxy. To test improper certificate ve
 
 In Burp, go to the `Proxy -> Options` tab, then go to the `Proxy Listeners` section, highlight your listener, and click `Edit`. Then go to the `Certificate` tab, check `Use a self-signed certificate`, and click `Ok`. Now, run your application. If you're able to see HTTPS traffic, your application is accepting self-signed certificates.
 
-- Accepting invalid certificates
+- Accepting possible rogue certificates
```

Comment:
I can't find a previous definition of "rogue certificates". Maybe define them previously in the section/chapter?